tests/integration/memory/game/component-test.js

```diff
@@ -0,0 +1,454 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, click, find } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import MemoryGameComponent from 'games-online/components/memory/game/component';
+
+module('integration | Component | memory/game', function (hooks) {
+  setupRenderingTest(hooks);
+
+  hooks.beforeEach(async function () {
+    this.sessionService = this.owner.lookup('service:session');
+    const store = this.owner.lookup('service:store');
+    const user = store.createRecord('user');
+
+    this.set('sessionService.currentUser', user);
+
+    class EnrichedMemoryGameComponent extends MemoryGameComponent {
+      shuffleCards() {
+        this.shuffledCards = this.cardArray;
+      }
+    }
+    this.owner.register('component:memory/game', EnrichedMemoryGameComponent);
+    await render(hbs`<Memory::Game />`);
+  });
+
+  test('validation test', async function (assert) {
+    assert.dom('[data-test-card-notStarted="0"]').hasNoAttribute('role');
+    assert.dom('[data-test-card-notStarted="1"]').hasNoAttribute('role');
+    assert.dom('[data-test-card-notStarted="2"]').hasNoAttribute('role');
+    assert.dom('[data-test-card-notStarted="3"]').hasNoAttribute('role');
+    assert.dom('[data-test-card-notStarted="4"]').hasNoAttribute('role');
+    assert.dom('[data-test-card-notStarted="5"]').hasNoAttribute('role');
+    assert.dom('[data-test-card-notStarted="6"]').hasNoAttribute('role');
+    assert.dom('[data-test-card-notStarted="7"]').hasNoAttribute('role');
+    assert.dom('[data-test-card-notStarted="8"]').hasNoAttribute('role');
+    assert.dom('[data-test-card-notStarted="9"]').hasNoAttribute('role');
+    assert.dom('[data-test-card-notStarted="10"]').hasNoAttribute('role');
+    assert.dom('[data-test-card-notStarted="11"]').hasNoAttribute('role');
```

Comment:
What do you think about:
```suggestion
    Array(12).fill('data-test-card-notStarted').map((selector, index) => assert.dom(`[`${selector}="${index}"`]`).hasNoAttribute('role'));
```

Reply:
there was a problem with super string, but I menaged to fix that

```js
Array(12)
      .fill('data-test-card-notStarted')
      .map((selector, index) =>
        assert.dom(`[${selector}="${index}"]`).hasNoAttribute('role')
      );
```